backend/EasySpeak.Core/EasySpeak.Core.DAL/Entities/Call.cs

```diff
@@ -6,11 +6,12 @@
 
 namespace EasySpeak.Core.DAL.Entities
 {
-    public class Call : Entity<long>
+    public class Call : Entity<long>, ICreatedBy
```

Comment:
Maybe it is better to create class, that extands Entity and has field CreatedBy? See the example https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-easyspeak/blob/80f430b33f73580294a08f4949ca759910dc7aa7/backend/EasySpeak.Core/EasySpeak.Core.DAL/Entities/Common/AuditEntity/AuditEntity.cs#L3